diagnostics_tools/include/diagnostic_tools/periodic_event_status.h

```diff
@@ -66,9 +66,25 @@ class PeriodicEventStatusParams final {
 
   const Diagnostic &stale_diagnostic() const { return stale_diagnostic_; }
 
+  PeriodicEventStatusParams &last_cycle_period_avg_window(double last_cycle_period_avg_window) {
+    last_cycle_period_avg_window_ = last_cycle_period_avg_window;
+    return *this;
+  }
+
+  double last_cycle_period_avg_window() const {return last_cycle_period_avg_window_;}
```

Comment:
@patapufate Hmm, in what units are window sizes in?

Reply:
@patapufate also, nit: 
```suggestion
  double last_cycle_period_avg_window() const { return last_cycle_period_avg_window_; }
```

Reply:
The Window sizes units are in double.